custom_components/tuya_local/devices/th08_temperature_and_humidity_sensor.yaml

```diff
@@ -0,0 +1,53 @@
+name: Temperature and Humidity Sensor
```

Comment:
According to the naming convention 
https://developers.home-assistant.io/blog/2022/07/10/entity_naming/
only the first word in device and entity names should be capitalised.